contrib/charts/cert-manager/templates/certificate-crd.yaml

```diff
@@ -11,8 +11,12 @@ metadata:
 spec:
   group: certmanager.k8s.io
   version: v1alpha1
+  scope: Namespaced
```

Comment:
This change is not meaningless, I wanted to prevent an extra newline generated by `toYaml` between the list and scope. It's a tradeoff between consistency with other templates, Helm Template best practices and keeping the release as small as possible.